docs/eventing/channels/subscriptions.md

```diff
@@ -24,32 +23,43 @@ After you have created a channel and a sink, you can create a subscription to en
+    - `--sink` specifies the target destination to which the event should be delivered.
+    By default, the `<sink_name>` is interpreted as a Knative service of this name, in the same
+    namespace as the subscription.
+    You can specify the type of the sink by using one of the following prefixes:
 
         - `ksvc`: A Knative service.
         - `svc`: A Kubernetes Service.
-        - `channel`: A channel that should be used as destination. Only default channel types can be referenced here.
+        - `channel`: A channel that should be used as the destination. You can only reference default
+        channel types here.
         - `broker`: An Eventing broker.
-    \
-    &nbsp;
-    - `--sink-reply` and `--sink-dead-letter` are optional arguments. They can be used to specify where the sink reply will be sent to, and where to send the cloud event in case of a failure, respectively. Both use the same naming conventions for specifying the sink as the `--sink` flag.
+        - `--sink-reply` is an optional argument you can use to specify where the sink reply is sent.
+        It uses the same naming conventions for specifying the sink as the `--sink` flag.
+        - `--sink-dead-letter` is an optional argument you can use to specify where to send the cloud
+        event in case of a failure. It uses the same naming conventions for specifying the sink as the
+        `--sink` flag.
 
-    Example command:
-    ```
-    kn subscription create mysubscription --channel mychannel --sink ksvc:myservice
-    ```
+            - `ksvc`: A Knative service.
+            - `svc`: A Kubernetes Service.
+            - `channel`: A channel that should be used as destination. Only default channel types can be referenced here.
+            - `broker`: An Eventing broker.
+        - `--sink-reply` and `--sink-dead-letter` are optional arguments. They can be used to specify where the sink reply will be sent to, and where to send the cloud event in case of a failure, respectively. Both use the same naming conventions for specifying the sink as the `--sink` flag.
 
-    This example command creates a channel named `mysubscription`, that routes events from a channel named `mychannel` to a Knative service named `myservice`.
-
-    **NOTE:** The sink prefix is optional. It is also possible to specify the service for `--sink` as just `--sink <service_name>` and omit the `ksvc` prefix.
+    This example command creates a channel named `mysubscription` that routes events from a channel
+    named `mychannel` to a Knative service named `myservice`.
 
+    !!! note
+        The sink prefix is optional. You can also specify the service for `--sink` as just
+        `--sink <service_name>` and omit the `ksvc` prefix.
```

Comment:
a few of the variables on this page use `<something_underscore>` instead of `<something-dash>` - can you update these please